src/facilities-context/actions/index.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  Facilities,
+  Facility,
+  RtDataMapping,
+} from "../../page-multi-facility/types";
+export * from "./rtData";
+export * from "./facilities";
+
+export type FacilitiesActions = {
+  type: FacilitiesActionTypes;
+  payload?: Partial<Facility> | Facilities | RtDataMapping | null;
+};
```

Comment:
Yeah I think it would be safer/better, I started with this just to get it going but I had also thought of changing it to have more explicit combinations. I'll update these!